### PR TITLE
fbinstant.get_sdk_version()

### PR DESCRIPTION
**Pull request by subsoap**

### CODE REVIEW

**Comment by subsoap** [comment]
Adds #30

File: fbinstant/utils/mock.lua [hunk 1]
```diff
@@ -115,6 +115,12 @@ function fbinstant.get_supported_apis()
 	return {}
 end
 
+function fbinstant.get_sdk_version()
+	print("get_sdk_version")
+	return "6.1"
```
Comment (AGulev) [inline fbinstant/utils/mock.lua]: FBInstant SDH has  getSDKVersion( ) method:
https://developers.facebook.com/docs/games/instant-games/sdk/fbinstant6.1
@britzl @subsoap

Comment (subsoap) [reply fbinstant/utils/mock.lua]: This it the mock api to use when access to the real api isn't available.

Comment (AGulev) [reply fbinstant/utils/mock.lua]: ok, sorry.

Comment (subsoap) [reply fbinstant/utils/mock.lua]: np!

**Pull request merged**